Packs/IntSight/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "IntSights",
     "description": "Use IntSights to manage and mitigate threats.",
     "support": "xsoar",
```

Comment:
is it going to stay xsoar supported? or are we changing ownership?

Reply:
Currently XSOAR supported